cscs-checks/libraries/math/scalapack_compile_run.py

```diff
@@ -27,12 +27,14 @@ def __init__(self, linkage):
         self.maintainers = ['CB', 'LM', 'MKr']
         self.tags = {'production'}
 
-    def setup(self, environ, partition, **job_opts):
-        super().setup(environ, partition, **job_opts)
+    def setup(self, partition, environ, **job_opts):
         if (self.current_system.name in ['kesch', 'monch'] and
-            self.current_environ.name == 'PrgEnv-gnu'):
+            environ.name == 'PrgEnv-gnu'):
+            #            self.modules += ['scalapack']
```

Comment:
@ajocksch @victorusu 
This is related depends on https://github.com/eth-cscs/production/issues/852.

However, I don't think that this test should be supported by PrgEnv-cray on Kesch. It has now meaning. What do you think?

Reply:
What about add `environ.name.startswith('PrgEnv-gnu'):` instead of `environ.name == 'PrgEnv-gnu'`?

Reply:
This test is only relevant for MPI-enabled environments.

Reply:
@victorusu What do you think about `PrgEnv-cray`? I am not sure it has any meaning.

Reply:
Ignore my last comment. The `PrgEnv-cray` uses libsci as Scalapack, so it is a supported environment.